src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Review/index.tsx

```diff
@@ -46,17 +46,18 @@ const ContractInteractionReview = ({ onClose, onPrev, tx }: Props): React.ReactE
   const dispatch = useDispatch()
   const { address: safeAddress } = useSelector(safeSelector) || {}
   const [gasCosts, setGasCosts] = useState('< 0.001')
-
+  const { nativeCoin } = getNetworkInfo()
```

Comment:
I think that you can move it outside the component and then won't be necessary to add the dependency in the useEffect hook.

Reply:
note that it's happening in several places in this PR.